src/components/Story/MainPanel/StoryMainPanelFooter/style.scss

```diff
@@ -0,0 +1,22 @@
+@use 'sass:math';
+@import '../../styles/variables';
+
+.ptr-StoryMainPanelFooter-default {
+	width: 100%;
+	height: 100%;
+	display: flex;
+	flex-flow: column;
+	justify-content: center;
+	align-items: center;
+	background-color: rgb(230, 243, 255);
+}
+
+.ptr-StoryMainPanelFooter-cure {
```

Comment:
CURE is a specific project. If there are some extensions of general story components for CURE, they should be part of CURE project. If 'cure' is just a name for advanced story usage, please rename it.

Reply:
renamed here [e337634](https://github.com/gisat-panther/visat-components/pull/28/commits/e3376348d84e2f82e86744a8387b42b4b047c0af)